.azure/pipelines/helix-matrix.yml

```diff
@@ -48,3 +52,41 @@ jobs:
     - name: Helix_logs
       path: artifacts/log/
       publishOnError: true
+      
+# Helix ARM64
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+  - template: jobs/default-build.yml
+    parameters:
+      jobName: Helix_matrix_arm64
+      jobDisplayName: "Tests: Helix ARM64 matrix"
+      agentOs: Windows
+      timeoutInMinutes: 480
+      steps:
+      - script: ./eng/build.cmd
+                -ci
+                -arch x64
+                -pack
+                -all
+        displayName: Build x64/arm64      
```

Comment:
Need `/p:CrossgenOutput=false` here too; might need it in the next build step as well